plugins/micromega/certificate.ml

```diff
@@ -748,6 +759,7 @@ let reduce_coprime psys =
   match oeq with
   | None -> None (* Nothing to do *)
   | Some ((v, n1, n2), (c1, p1), (c2, p2)) ->
+    Printf.printf "co_prime\n";
```

Comment:
Is this there on purpose? Looks like a debug statement.

Reply:
Indeed. Removed.